### DISCUSSION TITLE
Flipbook page-turn animation with "Slides per group skip"

### DISCUSSION THREAD

**Discussion by websirnik** [nolimits4web/swiper#7358] [Ideas]
Configuring the ['Slides per group skip'](https://swiperjs.com/demos#slides-per-group-skip) feature seems to work well for the flipbook use case. It would be even more flipbooky if it had a page-turn animation similar to the classic web flipbook. 

If we would like to implement this feature ourselves, where should we start? 
Also, we can sponsor the development of this feature if anyone is interested.

![2024-03-08 12-16-19 2024-03-08 12_17_44](https://github.com/nolimits4web/swiper/assets/282871/dac4575c-14da-41a7-bd82-9332879a42ac)

**Comment by sharon-alexander** [comment]
Would love this feature!
